### ISSUE TITLE
✅ 🐞 Bug Report: Global Widgets Don't Save - Elementor 3.13.2 - [ED-10855]

### ISSUE THREAD

**Issue by LoganStrike** [elementor/elementor#22544]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When using the right-click contextual menu to save a widget as global, the global widget doesn't save. The spinning arrow icon continue to spin, but the process never completes and the widget is not saved as a global.

### Steps to reproduce

Add a widget to a page. Right click, choose "Save as a global", enter a name, click Save. The widget doesn't get saved.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Flywheel/5.1.0
	MySQL version: (Google) v5.7.40
	PHP Version: 7.4.33.2
	PHP Memory Limit: 128M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 300M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: There are some writing permissions issues with the following directories/files:
		 - WordPress root directory
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.3
	Site URL: ***
	Home URL: ***
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 200M
	Max Memory limit: 200M
	Permalink Structure: /%category%/%postname%/
	Language: en-US
	Timezone: America/Los_Angeles
	Admin Email: ***
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.7.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.13.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.2
		Author: Elementor.com


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
PHP: showing 2 of 2PHP: 2022-05-19 15:45:59 [notice X 11][/www/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-19 16:06:51 [notice X 5][/www/wp-content/plugins/elementor/includes/managers/image.php::87] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2022-05-23 02:33:26 [error X 2][***/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.5:2:515479] Cannot read properties of null (reading \'getBoundingClientRect\') 

Log: showing 20 of 242023-05-22 21:01:08 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-22 21:01:08 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2023-05-22 21:01:08 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2023-05-22 21:01:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.13.2',
)]
2023-05-22 21:01:10 [info] elementor-pro::elementor_pro_updater Started 
2023-05-22 21:01:10 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-22 21:01:10 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-22 21:01:10 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Start  
2023-05-22 21:01:10 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Finished 
2023-05-22 21:01:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.1',
  'to' => '3.13.1',
)]
2023-05-24 12:27:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.2',
  'to' => '3.13.3',
)]
2023-05-24 12:27:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-05-24 12:27:18 [info] elementor::elementor_updater Started 
2023-05-24 12:27:18 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-24 12:27:18 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-24 12:27:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.2',
  'to' => '3.13.3',
)]
2023-05-24 12:27:22 [info] elementor-pro::elementor_pro_updater Started 
2023-05-24 12:27:22 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-24 12:27:22 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-24 12:27:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @LoganStrike! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hello there @LoganStrike!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.13.4! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
